.github/workflows/Packages - EdFi.Ods.Extensions.Homograph.yml

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Packages - EdFi.Ods.Extensions.Homograph
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Looking at the exitsing build in TeamCity, this only got triggered on main builds or manually, so please remove this pull request trigger. If you need to keep it here to confirm everything works at first, that is fine. But once complete with any testing work, this should be removed.

Reply:
Fixed it